### DISCUSSION TITLE
what's the refund policy?

### DISCUSSION THREAD

**Discussion by afxmedia** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#621] [General]
Hi, couldn't find this info on your website. 
I have two m1 MacBook pro 16" currently testing out using the free version. Not everything seems to be working. For instance, on one machine plugged in using the supplied MagSafe power adapter, it continues to charge despite me setting the charge limit to 60%. However, on the other machine using in clamshell mode supplied with usb-c power through a dock, it seems to work.

So I'm contemplating trying out the "Pro" features, but don't want to stuck with it if it's not working properly for me. I'm using the latest version 1.15.2.

@davidwernhart @MatthiasKerbl

**Comment by MatthiasKerbl** [comment]
Hi @afxmedia,

The Charge Limiter of AlDente Free should definitely also work with the MagSafe charger.

We have a 30-day money-back guarantee.

Cheers,
Matthias

**Comment by afxmedia** [comment]
Thanks @MatthiasKerbl  Sounds good. Is there any coupon/discount currently running?

**Reply by MatthiasKerbl** [reply]
Unfortunately, there is no discount available right now.

**Comment by afxmedia** [comment]
purchased! hope it works out.

**Reply by MatthiasKerbl** [reply]
Thanks a lot. :)
